docs/static/settings-file.asciidoc

```diff
@@ -178,6 +178,17 @@ Values other than `disabled` are currently considered BETA, and may produce unin
 | When set to `true`, quoted strings will process the following escape sequences: `\n` becomes a literal newline (ASCII 10). `\r` becomes a literal carriage return (ASCII 13). `\t` becomes a literal tab (ASCII 9). `\\` becomes a literal backslash `\`. `\"` becomes a literal double quotation mark. `\'` becomes a literal quotation mark.
 | `false`
 
+| `config.field_reference.escape_style`
+a| _EXPERIMENTAL_ setting that provides a way to reference fields that contain <<formal-grammar-escape-sequences,field reference special characters>> `[` and `]`.
+
+Current options are:
+
+* `PERCENT`: URI-style `%`{plus}`HH` hexadecimal encoding of UTF-8 bytes (`[` -> `%5B`; `]` -> `%5D`)
+* `AMPERSAND`: HTML-style `&#`{plus}`DD`{plus}`;` encoding of decimal Unicode code-points (`[` -> `&#91;`; `]` -> `&#91;`)
+* `NONE`: field names containing special characters _cannot_ be referenced.
+
+| `NONE`
```

Comment:
Is this entry case sensitive?  In this topic, we already have two instances of `None` and one instance of `none`.

Reply:
It is. And there really is no reason for it to be upcase, so I have changed it to be downcase to match the other `none`. 

Both other instances of "None" in the file should actually be "_N/A_" since they represent an _absence_ of a default value instead of a default value that is the literal `n`+`o`+`n`+`e`